### DISCUSSION TITLE
Add image from URL

### DISCUSSION THREAD

**Discussion by jacky4566** [inventree/InvenTree#4211] [Ideas]
A great feature would be adding images from URL. Saves us from downloading images then re-uploading them. 

I will probably work to implement this myself but not for another 3+ months.

**Comment by wolflu05** [comment]
Hello @jacky4566 

This is already there, you need to enable it via settings > server configuration > download from url

**Comment by jacky4566** [comment]
Oh. Well then. Clearly I didn't search well enough...
Perhaps this should be defaulted on? Is there a security concern?

**Reply by matmair** [reply]
There are indeed security concerns. We have it relatively well cornered off. A good chunk of our security advisories were related to this. The best practices are implemented now but in the end you still enable low privileged accounts to make your backend call *any* external address. This exposes information about your service and opens the door to a lot of things.
You need to gauge how much you trust your users and how advanced your attackers might be. It depends on your risk modell in the end.
